tests/store/api/test_store_import_func.py

```diff
@@ -250,12 +254,16 @@ def test_sync_microbial_orderform_dry_run(microbial_store: Store, microbial_orde
     )
 
     # THEN same number of active and inactive mic applications in status database
-    active_mic_apps_after_when = microbial_store.applications(
-        category=prep_category, archived=False
-    ).count()
-    inactive_mic_apps_after_when = microbial_store.applications(
-        category=prep_category, archived=True
-    ).count()
+    active_mic_apps_after_when = len(
```

Comment:
Took me a while to know what it was meant by when, maybe rename to this?
```suggestion
    active_mic_apps_after_sync: int = len(
```